server/routes/api/landing_page.py

```diff
@@ -41,20 +41,29 @@
 OVERVIEW = 'Overview'
 
 
-def get_landing_page_data(dcid, new_stat_vars):
-    return get_landing_page_data_helper(dcid, '^'.join(new_stat_vars))
+def get_landing_page_data(dcid, new_stat_vars, category,
+                          new_landing_page_cache_flag):
+    return get_landing_page_data_helper(dcid, '^'.join(new_stat_vars), category,
+                                        new_landing_page_cache_flag)
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_landing_page_data_helper(dcid, stat_vars_string):
+def get_landing_page_data_helper(dcid, stat_vars_string, category,
+                                 new_landing_page_cache_flag):
     data = {'place': dcid}
+    req_url = '/v1/internal/page/place/' + dcid
+    req_url += '?category={}&new_stat_vars={}'.format(
+        category, stat_vars_string.split(','))
     if stat_vars_string:
         data['newStatVars'] = stat_vars_string.split('^')
-    response = dc_service.fetch_data('/landing-page',
-                                     data,
-                                     compress=False,
-                                     post=True,
-                                     has_payload=False)
+    if new_landing_page_cache_flag == "false":
```

Comment:
I think we don't need this flag for this PR and can just change all the logic to use the new V1 API here. Having the flag won't help much, since if we decide to rollback, can just use and old version. It also requires removing the flag and clean the code later on.

Without that, you can also remove the cache_prefix, which makes the code simpler.

Reply:
Sorry I'd suggested that as a way of doing apples-to-apples performance comparison with and without this change on autopush instance.  Do you have a alternate suggestion?

Reply:
We could do this on dev - push current HEAD and send a bunch of requests and look at the API's performance stats, then push the new code and send the same requests. That will also take into account caching, will be isolated and simpler code wise. That could even be done without merging the PR.

Reply:
If this is for testing purpose, then it's fine. I am just a bit worried that removing the flag can introduce issues. 
Other option would be to deploy two instance before and post a a clean PR, but that would require more endprod work.

Reply:
Have cleaned it up and removed the flag. Also measured run times for the landing page data module for various cases and tabulated results: https://docs.google.com/spreadsheets/d/1NY6t5m2xY2Jc4EQ4V_8yLvrb2-N0cIEV8ItFREfYPsw/edit?usp=sharing